clients/admin-ui/src/features/common/Stepper.tsx

```diff
@@ -1,22 +1,56 @@
 import { Box, Stack, Text } from "@fidesui/react";
 import React from "react";
 
-import { StepperCircleIcon, VerticalLineIcon } from "./Icon";
+import {
+  StepperCircleCheckmarkIcon,
+  StepperCircleIcon,
+  VerticalLineIcon,
+} from "./Icon";
 
 interface Props {
   activeStep: number | null;
+  setActiveStep: Function;
   steps: { number: number; name: string }[];
 }
 
-const Stepper = ({ activeStep, steps }: Props) => (
+const Stepper = ({ activeStep, setActiveStep, steps }: Props) => (
   <Stack direction={["column", "row"]} w="100%">
     <Stack alignItems="center" direction="column" spacing={0}>
       {steps.map((step) => (
         <React.Fragment key={step.number}>
-          <StepperCircleIcon
-            boxSize={8}
-            stroke={activeStep === step.number ? "secondary.500" : "inherit"}
-          />
+          {activeStep &&
+          activeStep !== 1 &&
+          activeStep !== step.number &&
+          activeStep > step.number - 1 ? (
+            <StepperCircleCheckmarkIcon
+              boxSize={8}
+              cursor={
+                step.number === activeStep || step.number <= activeStep
```

Comment:
I personally feel like the active step should be `default` not `pointer` since clicking on it wouldn't do anything?

Reply:
That's fair! I'll make that change now.